apisix/plugins/ocsp-stapling.lua

```diff
@@ -0,0 +1,253 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements.  See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership.  The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License.  You may obtain a copy of the License at
+--
+--   http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied.  See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+--
+
+local require = require
+local get_request = require("resty.core.base").get_request
+local http = require("resty.http")
+local ngx = ngx
+local ngx_ocsp = require("ngx.ocsp")
+local ngx_ssl = require("ngx.ssl")
+local radixtree_sni = require("apisix.ssl.router.radixtree_sni")
+local core = require("apisix.core")
+local apisix_ssl = require("apisix.ssl")
+
+local plugin_name = "ocsp-stapling"
+local ocsp_resp_cache = ngx.shared[plugin_name]
+
+local plugin_schema = {
+    type = "object",
+    properties = {},
+}
+
+local _M = {
+    name = plugin_name,
+    schema = plugin_schema,
+    version = 0.1,
+    priority = -44,
```

Comment:
The priority should be positive integer

Reply:
the behavior of this plugin is close to the GM plugin, so I have positioned it after `-43`.